index.html

```diff
@@ -263,6 +160,21 @@ <h2 class="neonText2">Skills</h2>
         <img src="images/Shape.png" alt="">
     </footer>
 
+    <a id="testing" href="#" class="neonText2">
+        <div class="blind">PwD</div>
+    </a>
+
+    <section class="modals-section">
+        <div id="modal-containerID0" class="modal-container">
+            <div id="modalID" class="modal">
+
+            </div>
+        </div>
+    </section>
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure the modal on the mobile version has the `see live button` and `see source button` to match the Figma design  below:






*figma*
![figma_modal](https://user-images.githubusercontent.com/59397159/173973047-7e5a9345-58bf-4a74-b4c5-8f83d0e3f493.png)

*your's*
![mobile_modal](https://user-images.githubusercontent.com/59397159/173973050-55bf7717-1084-492c-8f63-d7ba98fe74b5.png)

Reply:
Could you kindly check it again, please.
I'm gonna try to make some more changes on it but this is how it shows here on my inspecetion section, so as on my phone when I test it.

![CodeReviews](https://user-images.githubusercontent.com/103270117/173975637-18c49c48-cb77-41bc-87c8-7f5776f04d9f.png)

Reply:
- This issue is not yet resolved :x: